service/routes.py

```diff
@@ -53,15 +53,21 @@ def list_inventory():
 ######################################################################
 # RETRIEVE AN INVENTORY ITEM
 ######################################################################
-@app.route("/inventory/<int:item_id>", methods=["GET"])
-def get_inventory(item_id):
+@app.route("/inventory/pid/<int:pid>/condition/<int:condition_id>", methods=["GET"])
+def get_inventory(pid, condition_id):
     """
     Retrieve a single inventory item
 
     This endpoint will return an inventory item based on it's id
     """
-    app.logger.info("Request for inventory item with id: %s", item_id)
-    return {}, status.HTTP_200_OK
+
+    app.logger.info("Request for inventory item with id: %s", pid)
+    inventory = Inventory.find_by_pid_condition(pid, Condition(condition_id))
```

Comment:
could you check if condition_id is valid or not similar to :

https://github.com/DevOps-Fall2022-Inventory/inventory/blob/master/service/routes.py#L96